topcoffea/modules/corrections.py

```diff
@@ -58,6 +59,7 @@ def get_method1a_wgt_doublewp(effA, effB, sfA, sfB, cutA, cutB, cutC):
 def btag_sf_eval(jet_collection,wp,year,method,syst):
     # Get the right sf json for the given year
 
+    dt_era = "Run3" if year[2] == "2" else "Run2"
```

Comment:
I would suggest replacing the strings with a bool `is_run3` to simplify statements later on.  That way we don't need to check for string agreement, and we only need one variable (since we can just do e.g., `if is_run3:` or whatever).

Reply:
Sounds good!